quipucords/tests/api/aggregate_report/test_aggregate_report_model.py

```diff
@@ -38,9 +88,13 @@ def report_and_expected_aggregate() -> tuple[Report, AggregateReport]:  # noqa:
     Please read the expected_aggregate updates between objects carefully
     to understand how each new object should affect the end result.
     """
-    expected_aggregate = AggregateReport()
+    expected_aggregate = expected_aggregate_template().copy()
```

Comment:
while this works, the "pytest recommended" approach would be "expect_aggregate" to be its own fixture